app/src/main/java/org/thoughtcrime/securesms/conversation/v2/messages/VisibleMessageView.kt

```diff
@@ -190,7 +190,14 @@ class VisibleMessageView : LinearLayout {
         binding.dateBreakTextView.text = if (showDateBreak) DateUtils.getDisplayFormattedTimeSpanString(context, Locale.getDefault(), message.timestamp) else null
         binding.dateBreakTextView.isVisible = showDateBreak
         // Message status indicator
-        val (iconID, iconColor) = getMessageStatusImage(message)
+        val (iconID, iconColor, textId) = getMessageStatusImage(message)
+        if (textId != null) {
+            binding.messageStatusTextView.setText(textId)
```

Comment:
we might have to set the text to null in the null case as well so that re-binds that should have a null message status don't keep the old value (as they won't be set this way)

Reply:
it might not be an issue if we are controlling the visibility so that this case should never happen though I guess

Reply:
I updated the visibility logic so if the `textId` or `iconID` are null then the UI elements will be hidden to prevent the chance that stale data will be shown - let me know if you also want be to null-out the data